db/seeds.rb

```diff
@@ -65,9 +74,29 @@ def create_description(project_media)
   Media.last.type == "Link" ? humanize_link(Media.find(project_media.media_id).url) : Faker::Company.catch_phrase
 end
 
-def add_claim_descriptions_and_fact_checks(user, project_medias)
-  claim_descriptions = project_medias.map { |project_media| ClaimDescription.create!(description: create_description(project_media), context: Faker::Lorem.sentence, user: user, project_media: project_media) }
-  claim_descriptions.values_at(0,3,8).each { |claim_description| FactCheck.create!(summary: Faker::Company.catch_phrase, title: Faker::Company.name, user: user, claim_description: claim_description) }
+def create_claim_descriptions(user, project_medias)
+  project_medias.map { |project_media| ClaimDescription.create!(description: create_description(project_media), context: Faker::Lorem.sentence, user: user, project_media: project_media) }
+end
+
+def create_fact_checks(user, claim_descriptions)
+  claim_descriptions.each { |claim_description| FactCheck.create!(summary: Faker::Company.catch_phrase, title: Faker::Company.name, user: user, claim_description: claim_description, language: 'en') }
+end
+
+def verify_fact_check_and_publish_report(project_medias)
+  project_medias[0..9].each do |pm|
+    status = ['verified', 'false'].sample
+
+    verification_status = pm.get_dynamic_annotation('verification_status')
+    verification_status.set_fields = { verification_status_status: status }.to_json
+    verification_status.reload
+    verification_status.save!
```

Comment:
For simplicity we can use this code to update status
```
verification_status = pm.last_status_obj
verification_status.status = status
verification_status.save!
```

Reply:
thanks @melsawy ❤️